### PR TITLE
Eventoverview bookedeventoverview

### PR DESCRIPTION
**Pull request by Kailiangdong**
## Issue

This fixes the following issue(s):
I make the UI for Events（booked events) overview for our new Event data type.
I only reuse the layout of Filmcard but did not reuse Card class. And there is a button which can switch between bookedevent and all event.

The EventsController given in #853 currently provides static Event (booked event)objects to test the new fragments. In later commit  we will connect the EventDetailsActivity in #853 with Eventsactivity.

## Screenshot
image
image

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsAdapter.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        TextView locality;
+        TextView srcDate;
+
+        public ViewHolder(View view) {
+            super(view);
+            cardView = (CardView) view;
+            title = (TextView) view.findViewById(R.id.events_title);
+            img = (ImageView) view.findViewById(R.id.events_img);
+            locality = (TextView) view.findViewById(R.id.events_src_locality);
+            srcDate = (TextView) view.findViewById(R.id.events_src_date);
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsAdapter.java]: fixed

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsAdapter.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    private static final Pattern COMPILE = Pattern.compile("^[0-9]+\\. [0-9]+\\. [0-9]+:[ ]*");
+    private final List<Event> meventlist;
+    private Context mContext;
+
+    static class ViewHolder extends RecyclerView.ViewHolder {
+        CardView cardView;
+        ImageView img;
+        TextView title;
+        TextView locality;
+        TextView srcDate;
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsAdapter.java]: fixed

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+        // Gets all news from mockup
+        events = EventsController.getEvents();
+
+        if (events.size() > 0) {
+            EventsAdapter adapter = new EventsAdapter(events);
+            lv = findViewById(R.id.activity_events_list_view);
+            lv.setLayoutManager(new LinearLayoutManager(this));
+            lv.setAdapter(adapter);
+        }
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java]: fixed

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+        this.mBookedShowMode = Utils.getSettingBool(this, Const.EVENT_BOOKED_MODE, false);
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java]: fixed

File: app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsActivity.java]: fixed

File: app/src/main/res/layout/card_events_item.xml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                android:layout_above="@+id/linearLayout"
+                android:layout_alignParentEnd="true"
+                android:layout_alignParentStart="true"
+                android:layout_alignParentTop="true"
+                android:layout_margin="10dp"
+                android:text="@string/info_title"
+                android:textSize="20sp" />
+
+            <LinearLayout
+                android:id="@id/linearLayout"
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/res/layout/card_events_item.xml]: fixed

File: app/src/main/res/layout/activity_events.xml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                    android:id="@+id/activity_events_list_view"
+                    android:layout_width="fill_parent"
+                    android:layout_height="fill_parent"
+                    android:scrollbars="vertical" />
+        </LinearLayout>
+
+    </LinearLayout>
+    <!-- The navigation drawer -->
+    <include layout="@layout/navigation_drawer" />
+</android.support.v4.widget.DrawerLayout>
```
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/res/layout/activity_events.xml]: Did't find source of problem. Merge first ,find later~

File: app/src/main/res/layout/card_events_item.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Kailiangdong) [reply app/src/main/res/layout/card_events_item.xml]: fixed

File: app/src/main/res/layout/activity_events.xml (hunk 6 above)
[the remark this answers is not in the document]
Comment (kordianbruck) [reply app/src/main/res/layout/activity_events.xml]: Please open a bug issue, so we don't forget.

[the remark this answers is not in the document]
Comment (schulzalexander) [reply app/src/main/res/layout/activity_events.xml]: just created the bug issue, it's #877

**Pull request merged**